monai/losses/dice.py

```diff
@@ -89,6 +99,7 @@ def __init__(
                 of the sequence should be the same as the number of classes. If not ``include_background``,
                 the number of classes should not include the background category class 0).
                 The value/values should be no less than 0. Defaults to None.
+            soft_label: whether the target contains non-binary values or not
```

Comment:
```suggestion
            soft_label: whether the target contains non-binary values (soft labels) or not. If True a soft label formulation of the loss will be used.
```
This clarifies a little bit I feel, the same should be done with the other modified losses.